modules/datadog-configuration/modules/datadog_keys/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.0"
+    }
```

Comment:
tflint raised these failures and would not allow committing for client without this addition.

Usually we always have a versions.tf in all consumable modules so to stay consistent id imagine that account map should have pins instead of this module omitting pins

Reply:
@nitrocode I'm with you on appeasing tflint, I made this argument in an earlier PR in this repo but closed it. Please see #635 though, this change breaks an upstream consumer of this component. I don't know the best solution, just want to bring this up.

Reply:
Sorry I was wrong, it's not a "versions.tf" problem, it's a `provider {}` problem. Trying to find the source now.